### PR TITLE
Add option for boundscontrol to scale about object center

### PR DESCRIPTION
**Pull request by camnewnham**
## Overview
This PR adds the option between scaling around the opposite corner (existing and default behaviour) and the bounds center.

## Changes
- Add: `ScaleAnchorType` enum
- Add: `scaleAnchor ` serializable property on `BoundsControl`
- Add: Logic to select the anchor between the opposite corner or bounds center.

### CODE REVIEW

File: com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs [hunk 1]
```diff
@@ -278,6 +278,26 @@ public RotateAnchorType RotateAnchor
...
+        [SerializeField]
+        [Tooltip("Specifies whether the scale handles will rotate the object around their opposing corner, or the center of its calculated bounds.")]
+        private ScaleAnchorType scaleAnchor = ScaleAnchorType.OppositeCorner;
+
+        /// <summary>
+        /// Specifies whether the scale handles will rotate the scale around the opposing corner, or the center of its calculated bounds.
+        /// </summary>
+        public ScaleAnchorType ScaleAnchor
+        {
+            get { return scaleAnchor; }
```
Comment (Zee2) [inline com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs:291]: Format nit, `get => scaleAnchor;`

**Comment by Zee2** [comment]
@camnewnham , thanks for this excellent PR! The formatting and readability is great and we really appreciate your contribution. I'll take a look at this locally soon and give a review.

File: com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs (hunk 1 above)
Comment (camnewnham) [reply com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs]: Agreed - I copied the style from `RotateAnchor` but will update that and `ScaleBehaviour`.

**Comment by Zee2** [comment]
Looks great! Thanks again for the contribution @camnewnham.

**Review by Zee2 — APPROVED** [review]

**Pull request merged**